week_4/quiz-trivia-app/src/app/app.config.ts

```diff
@@ -0,0 +1,11 @@
+import { ApplicationConfig, provideZoneChangeDetection } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideZoneChangeDetection({ eventCoalescing: true }),
```

Comment:
ChangeDetectionStrategy.OnPush on component level can help you with component change detection. If you dont need a global change detection please uninstall zone js and this and use the onPush strategy on component level

Reply:
Zone js wasn't install